classes/security/authorization/internal/SubmissionFileStageRequiredPolicy.inc.php

```diff
@@ -34,17 +42,21 @@ function __construct($request, $fileIdAndRevision = null) {
 	function effect() {
 		$request = $this->getRequest();
 
+		// Get the user
+		$user = $request->getUser();
+		if (!is_a($user, 'PKPUser')) return AUTHORIZATION_DENY;
```

Comment:
This is probably a hold-over from a previous class that was used as a template for this one. Suggest removing this check.